examples/server/server.cpp

```diff
@@ -49,10 +49,10 @@ static bool server_verbose = false;
 #define LOG_VERBOSE(MSG, ...)                                            \
     do                                                                   \
     {                                                                    \
-        if (server_verbose)                                              \
-        {                                                                \
-            server_log("VERBOSE", __func__, __LINE__, MSG, __VA_ARGS__); \
-        }                                                                \
+	if (server_verbose)                                              \
+	{                                                                \
+	    server_log("VERBOSE", __func__, __LINE__, MSG, __VA_ARGS__); \
+	}                                                                \
```

Comment:
This project uses spaces for indentation. Do not replace them with tabs.

Reply:
fixed.  my editor defaults to tab compression on write.